static/includes/MigratingVMs.md

```diff
@@ -28,26 +28,20 @@ To migrate VMs from TrueNAS 24.10 (latest) to 25.04:
    - Note the **Zvol path**, storage **Mode**, and **Disk Sector Size**.
    - Document all VM devices and their configurations for later use.
 
-3. Locate or Download the OS Image File
-   - Access the VM via **Display** or **Serial Shell** to confirm the installed OS and version.
-      Refer to documentation for the installed OS, if needed.
-   - Check if the **ISO file** used for installation is stored on TrueNAS or in an external location.
-   - If needed, download a matching OS image before proceeding.
-
 #### Recreating VMs in 25.04
 
 1. Configure Virtualization Settings
-   - After upgrading to TrueNAS 25.04, go to **Instances**.
+   - After upgrading to TrueNAS 25.04, go to **Instances** (formerly **Virtualization**).
    - Click **Select Pool** to open **Global Settings**.
    - Use the **Pool** dropdown to select a storage pool for virtualization.
    - Accept default networking settings or modify as needed, then click **Save**.
```

Comment:
```suggestion
   a. After upgrading to TrueNAS 25.04, go to **Instances** (formerly **Virtualization**).
   b. Click **Select Pool** to open **Global Settings**.
   c. Use the **Pool** dropdown to select a storage pool for virtualization.
   d. Accept default networking settings or modify as needed, then click **Save**.
```
Correct documentation of procedures is to use ordered lists nested under the primary when the steps need to be performed in a particular sequence.